data/transactions/logic/fields.go

```diff
@@ -236,21 +237,34 @@ func (s tfNameSpecMap) get(name string) (FieldSpec, bool) {
 	return fs, ok
 }
 
+// ItxnVersion is an interface to help with doc generation
+// that lets us check what version certain txn fields were
+// allowed to be used in inner transactions
+type ItxnVersion interface {
+	ItxnVersion() uint64
+}
```

Comment:
Added this as an interface so the `opdoc` cmd could ask what version a txn field was available in inner transactions